tests/acceptance/features/bootstrap/SpacesContext.php

```diff
@@ -3134,4 +3134,33 @@ public function publicDownloadsTheFolderFromTheLastCreatedPublicLink(string $res
 			)
 		);
 	}
+
+	/**
+	 * @Then /^the json responded should contain these key and value pairs:$/
+	 *
+	 * Loops through data items in response to find given key and value pairs
+	 * It does not support field value replacing
+	 *
+	 * @param TableNode $table
+	 *
+	 * @return void
+	 * @throws Exception
+	 */
+	public function theJsonRespondedShouldContainTheseKeyAndValuePairs(TableNode $table):void {
+		$this->featureContext->verifyTableNodeColumns($table, ['key', 'value']);
+		$response = json_decode((string)$this->featureContext->getResponse()->getBody(), true, 512, JSON_THROW_ON_ERROR);
```

Comment:
we do have function for that
```suggestion
		$response = $this->featureContext->getJsonDecodedResponse($this->featureContext->getResponse())
```